app/views/components/_publisher-metadata.html.erb

```diff
@@ -18,7 +18,8 @@
             values = Array(values)
           %>
           <% if values.any? %>
-            <dt class="app-c-publisher-metadata__term"><%= title %>: </dt>
+            <% metadata_id = values.to_s =~ /href/ ?  "id=metadata-" + title.to_s.parameterize : "" %>
```

Comment:
I think the need to do this reinforces the earlier point that we need to bring the attribute-adding-code nearer to the component ERB template.